lib/Frontend/CompilerInvocation.cpp

```diff
@@ -262,6 +262,30 @@ void CompilerInvocation::setSDKPath(const std::string &Path) {
   updateRuntimeLibraryPaths(SearchPathOpts, LangOpts.Target);
 }
 
+void CompilerInvocation::setModuleAliasMap(std::vector<std::string> args) {
```

Comment:
Could we factor out the common code so that `ArgsToFrontendOptionsConverter::computeModuleAliases` and swift-ide-test can share the implementation?  That would also let swift-ide-test get the same validation behaviour (you can create a PrintingDiagnosticConsumer + DiagnosticsEngine if we don't have one handy).

Reply:
Moved the compute func into its own namespace ModuleAliasesConverter